pkgs/applications/virtualization/qemu/default.nix

```
@@ -86,6 +86,14 @@ stdenv.mkDerivation rec {
   patches = [
     ./fix-qemu-ga.patch
     ./9p-ignore-noatime.patch
+    # Fix qemu writing `smb.conf` incorrectly, breaking
+    # `-net user,smb=/dir` mounts on samba >= 2.0.5.
+    # Remove when https://lists.gnu.org/archive/html/qemu-devel/2021-02/msg07156.html is merged and available
+    (fetchpatch {
+      name = "net-slirp-Fix-incorrect-permissions-on-samba-2.0.5.patch";
+      url = "https://github.com/nh2/qemu/commit/de30898a738bd073593d20930496ef63e54d62a3.patch";
```

```suggestion
      url = "https://github.com/qemu/qemu/commit/de30898a738bd073593d20930496ef63e54d62a3.patch";
```

makes it easier to track when it is merged.